app/Domain/Integrations/Controllers/IntegrationController.php

```diff
@@ -104,8 +104,23 @@ public function index(Request $request): Response
         ]);
     }
 
-    public function create(): Response
+    public function create(Request $request): RedirectResponse|Response
     {
+        $type = $request->query('type', '');
+
+        $integrationType = IntegrationType::tryFrom(
+            is_array($type) ? $type[0] : $type
```

Comment:
I wonder why this can be an array or a string?

Reply:
e.g.

`?type=entry -> string`

or

`?type=entry&type=search -> string[]`

Reply:
Yes, but it's a query parameter so some frontend code sets it as a string and other parts set it as an array?

Reply:
A user can manually set a url. It is quite common in frontend frameworks aswel that you handle both cases